node/network/src/message.rs

```diff
@@ -45,6 +46,8 @@ pub enum Message {
 
     /// Nightshade gossip.
     Gossip(Box<Gossip>),
+    /// Transaction gossip
+    TxGossip(Box<TxGossip>),
     /// Announce of tx/receipts between authorities.
     PayloadAnnounce(ChainPayload),
```

Comment:
Should remove payload announce then - that was suppose to be to send each other payloads.